src/Phinx/Console/PhinxApplication.php

```diff
@@ -45,12 +45,14 @@ class PhinxApplication extends Application
      *
      * Initialize the Phinx console application.
      *
-     * @param string $version The Application Version
+     * @param string $version The Application Version, if null, use version out of composer.json file
```

Comment:
string|null

Reply:
Why is this part of the change anyway?

Reply:
Sorry, I had just added this separately and was going to put this in a separate PR as I'm a bit confused as to what was the full intention here as it used to be that the actual version string was passed to the function, but then it was changed to load this out of composer.json file, but all of the tests still want to pass in a variable when initializing the class.

I can revert the change to the file here and then just open an issue with the above to get an idea as to what the desired functionality would be.

Reply:
I think you can fix up the doc block and that should be fine.

Reply:
Done